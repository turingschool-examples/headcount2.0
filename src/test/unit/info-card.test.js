import React from 'react';
import ReactDom from 'react-dom';
import { shallow } from 'enzyme';

import InfoCard from '../../info-card';

describe('InfoCard', () => {

  it('should have hold the right elements', () => {
    const wrapper = shallow(
                      <InfoCard
                        data=
                          {
                            { stats:
                               { '2004': 0.24,
                                 '2005': 0.278,
                                 '2006': 0.337,
                                 '2007': 0.395,
                                 '2008': 0.536,
                                 '2009': 0.598,
                                 '2010': 0.64,
                                 '2011': 0.672,
                                 '2012': 0.695,
                                 '2013': 0.703,
                                 '2014': 0.741 },
                              location: 'Colorado' 
                            } 
                          }
                      />);
    expect(wrapper).toMatchSnapshot()
  })

})