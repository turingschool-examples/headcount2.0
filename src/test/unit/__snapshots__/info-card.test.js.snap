// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoCard should have hold the right elements 1`] = `
<div
  className="info-card"
>
  <h4>
    Colorado
  </h4>
  <div
    className="year-list"
  >
    <p
      className="low"
      key="1538613777840"
    >
      2004
      : 
      0.24
      <span
        className="emoji"
      >
        ...驕좶잺
      </span>
    </p>
    <p
      className="low"
      key="1538613777857"
    >
      2005
      : 
      0.278
      <span
        className="emoji"
      >
        ...驕좶잺
      </span>
    </p>
    <p
      className="low"
      key="1538613777874"
    >
      2006
      : 
      0.337
      <span
        className="emoji"
      >
        ...驕좶잺
      </span>
    </p>
    <p
      className="low"
      key="1538613777891"
    >
      2007
      : 
      0.395
      <span
        className="emoji"
      >
        ...驕좶잺
      </span>
    </p>
    <p
      className="hi"
      key="1538613777892"
    >
      2008
      : 
      0.536
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777905"
    >
      2009
      : 
      0.598
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777918"
    >
      2010
      : 
      0.64
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777931"
    >
      2011
      : 
      0.672
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777944"
    >
      2012
      : 
      0.695
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777957"
    >
      2013
      : 
      0.703
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
    <p
      className="hi"
      key="1538613777970"
    >
      2014
      : 
      0.741
      <span
        className="emoji"
      >
        ...游때
      </span>
    </p>
  </div>
</div>
`;
